### PR TITLE
Bug 1756096: Included must-gather script and image

### PR DESCRIPTION
**Pull request by huffmanca**
This PR adds a script and corresponding image to work with `oc adm must-gather`. This will cycle through all of the CRDs and output any resources for them.

### CODE REVIEW

File: must-gather/gather [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      /usr/bin/oc get ${resource} -o yaml ${ocobject} > ${object_collection_path}/${ocobject}.yaml
+    else
+      object_collection_path=must-gather/namespaces/${ocns}/crs/${resource}
+      mkdir -p ${object_collection_path}
+      /usr/bin/oc get ${resource} -n ${ocns} -o yaml ${ocobject} > ${object_collection_path}/${ocobject}.yaml
+    fi
+  done
+done
+
+exit 0
```
Comment (gnufied) [inline must-gather/gather:26]: Can you add a quick doc about how to use must-gather with LSO? I would assume we will need more detailed docs in openshift-docs

Comment (huffmanca) [reply must-gather/gather:26]: I've included docs in a separate commit, and referenced this from the main README. I've included the current image name, and will update it if this is modified.

Comment (gnufied) [reply must-gather/gather:26]: thank you. one more question - does this supplement image needs to be built and shipped by ART pipeline?

Comment (huffmanca) [reply must-gather/gather:26]: It won't automatically get picked up. We'll need to file JIRAs to get ART to build it.

**Pull request merged**